### PR TITLE
Issue 2182 - Anax s390x support feature

### LINKED ISSUE #2182: Anax s390x support feature
s390x (mainframes) architecture is not supported so far as agent runtime host platform.

This issue was created to take up support dev process for s390x platform.

As base OS stack it supposed to use RHEL7.x/8.x.

Procedures to check out:
- Simple executable (anax) build, RPM packages (horizon and horizon-cli) build, Docker container build
- System service runtime
- Docker container runtime

### PR DESCRIPTION
**Pull request by jacob-rogers**
Signed-off-by: Dmitry Gorbachev <dmitry.gorbachev@ru.ibm.com>

Resolves #2182

### CODE REVIEW

**Review by bmpotter — COMMENTED** [review]
Please enumerate what tests you ran with the s390x edge node with an IEAM management hub.

**Comment by jacob-rogers** [comment]
@bmpotter Hi Bruce, here is the log of our test run 
[anax-e2edev-z.log](https://github.com/open-horizon/anax/files/5664516/anax-e2edev-z.log)
Command to verify:
```sh
make test-remote TEST_VARS="NOLOOP=1 NOCANCEL=1 NOKUBE=1"
```

File: anax-in-container/config/anax.json [hunk 1]
```diff
@@ -26,6 +26,8 @@
   "ArchSynonyms": {
     "x86_64": "amd64",
     "armhf": "arm",
-    "aarch64": "arm64"
+    "aarch64": "arm64",
+    "ppc64le": "ppc64le",
+    "s390x": "s390x"
```
Comment (bmpotter) [inline anax-in-container/config/anax.json:31]: These 2 lines are unnecessary. You only need to list synonyms that are different from the default name of the architecture.

**Review by bmpotter — CHANGES_REQUESTED** [review]

File: anax-in-container/config/anax.json (hunk 1 above)
Comment (jacob-rogers) [reply anax-in-container/config/anax.json]: Thanks, removed

**Review by bmpotter — APPROVED** [review]

**Pull request merged**